### PR TITLE
Meenakshi

### PR DESCRIPTION
**Pull request by Y-Meenakshi**
Please check the new pull request with the new API pages for the primary channel and files folder properties.

### CODE REVIEW

File: api-reference/beta/api/channel-get-filesFolder.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
```
Comment (Lauragra) [inline api-reference/beta/api/channel-get-filesFolder.md:1]: Global: Please make all the file names all lowercase, not mixed case.

**Comment by Lauragra** [comment]
@Y-Meenakshi  what is the merge status of this PR? Please take care of the merge conflicts. Why are we adding the files to both beta and v1.0 at the same time?

File: api-reference/beta/api/channel-get-filesFolder.md (hunk 1 above)
Comment (Y-Meenakshi) [reply api-reference/beta/api/channel-get-filesFolder.md:1]: Hi Laura, Il change the file naming into all lowercase.

**Comment by Y-Meenakshi** [comment]
Hi Laura, I have resolved the merge conflicts. The new API pages were created for the V1.0 version, now I added them in beta.

**Comment by Lauragra** [comment]
> Hi Laura, I have resolved the merge conflicts. The new API pages were created for the V1.0 version, now I added them in beta.

To be clear, did we ship APIs to beta without documentation? Generally we publish docs for beta, and then later we promote those APIs to v1.0. We don't generally ship APIs to both beta and v1.0 at the same time. @nkramer , can you help to clarify? Thanks.

**Pull request closed without merging**